server/form-pages/apply/schema.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://json-schema.org/draft/2019-09/schema",
   "type": "object",
   "title": "Apply Schema",
+  "additionalProperties": false,
```

Comment:
I think it would be best if you saved this change without formatting. It'll be easier to see this is the only relevant change (I think?) and when we come to generate the next schema it'll be easier to see what the changes area

Reply:
Ah, good catch. That was me just editing it in VS Code and the Prettier plugin doing its magic!